test/validation/api/classification/odp_classification_test_pmr.c

```diff
@@ -2477,6 +2522,20 @@ static int check_capa_pmr_series(void)
 	return support;
 }
 
+static int check_capa_pmr_marking(void)
+{
+	uint64_t terms;
+
+	terms = cls_capa.supported_terms.bit.dip_addr &&
+		cls_capa.supported_terms.bit.udp_dport;
+
+	/* one PMR for IP, MAX_NUM_UDP PMRs for UDP */
+	if (terms && cls_capa.max_mark >= (MAX_NUM_UDP + 1))
```

Comment:
The comparison should be with MARK_UDP + MAX_NUM_UDP - 1.

Reply:
fixed in v2